sympy/solvers/tests/test_solveset.py

```diff
@@ -1338,14 +1338,14 @@ def test_issue_5132_1():
 
     n = Dummy('n')
     eqs = [exp(x)**2 - sin(y) + z**2, 1/exp(y) - 3]
-    s_real_y = -log(3)
+    s_real_y = log(S(1)/3)
     s_real_z = sqrt(-exp(2*x) - sin(log(3)))
     soln_real = FiniteSet((s_real_y, s_real_z), (s_real_y, -s_real_z))
-    lam = Lambda(n, 2*n*I*pi + Mod(-log(3), 2*I*pi))
+    lam = Lambda(n, 2*n*I*pi + Mod(log(S(1)/3), 2*I*pi))
     s_complex_y = ImageSet(lam, S.Integers)
-    lam = Lambda(n, sqrt(-exp(2*x) + sin(2*n*I*pi + Mod(-log(3), 2*I*pi))))
+    lam = Lambda(n, sqrt(-exp(2*x) + sin(2*n*I*pi + Mod(log(S(1)/3), 2*I*pi))))
```

Comment:
It seems that this change should not be done (maybe because it is deep in the expression tree under the `sqrt` and `sin` -nodes).